packages/forms/src/validate.js

```diff
@@ -0,0 +1,10 @@
+import Ajv from 'ajv';
+
+export default function validate(jsonSchema, data) {
+	if (!data || !jsonSchema) {
+		return false;
+	}
+	const validator = new Ajv().compile(jsonSchema);
+	const valid = validator(data);
```

Comment:
could you be more defensive about validator that might be `null` according to the lib implementation ?

Reply:
I don't understand null is falsy so it should not be a problem here.
you want a `return !!valid` ?

Reply:
Validator may be valued to undefined by ajv library

Reply:
So `return validator && validator(data);` could do a better job, I agree.